src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -587,8 +586,13 @@ private void ParsePatchElement(XElement patchElement, out bool upToDate, out boo
                 var contentsAttribute = element.Attribute(ContentAttributeName);
                 if (contentsAttribute == null)
                 {
-                    _service._reportAndSwallowException(
```

Comment:
@sharwell In practice `_reportAndSwallowException` is actually `FatalError.ReportWithoutCrash` so I'll address my response to your other question.